source/includes/pipeline_config/_001-parameter.md.erb

```diff
@@ -0,0 +1,19 @@
+<%
+json ='{
+  "params": [
+    {
+      "name": "command",
+      "value": "echo"
+    }
+  ]
+}'
```

Comment:
instead of it being a sting, this could just be a ruby hash or array. Your `describe_sub_object` helper or template could then convert it to JSON.

The reason being that this'll get messy once you have special characters that need escaping because of the `json` being a string :)